bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WorkbookEditorsHandler.java

```diff
@@ -163,7 +163,7 @@ private List<EditorReference> getParts(WorkbenchPage page) {
 	private Map<EditorReference, String> generateColumnLabelTexts(List<EditorReference> editorReferences) {
 		Map<EditorReference, String> editorReferenceLabelTexts = new HashMap<>(editorReferences.size());
 		Map<String, List<EditorReference>> collisionsMap = editorReferences.stream()
-				.collect(Collectors.groupingBy(r -> r.getName()));
+				.collect(Collectors.groupingBy(r -> r.getTitle()));
```

Comment:
How about replacing null with empty string?

Reply:
`getTitle()` already does that and does not return null

Reply:
that can return descriptor.getLabel() that can return configurationElement.getAttribute(IWorkbenchRegistryConstants.ATT_NAME); that can return null

Reply:
True, I have added another Util.safeString() guard around it.